README.md

```diff
@@ -43,22 +110,39 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
-You will need to install ``mpi4py`` in your python environment. The install instructions
-can be found by selecting [mpi4py docs](https://mpi4py.readthedocs.io/en/stable/install.html).
+> Note that if you are using a Cray machine, you will need to run `CC=cc CXX=CC pip install -r requirements.txt`.
 
-Lastly, the C++ applications themselves need to be built. One CMake edit is required.
-Near the top of the CMake file, change the path to the ``SMARTREDIS`` variable to
-the top level of the directory where you built or installed the SmartRedis library.
+Lastly, the C++ applications themselves need to be built. To complete this, 
+one CMake edit is required. When running `cmake ..`, 
+change the path to the ``SMARTREDIS`` variable to the top level of the directory 
+where you built or installed the SmartRedis library using the ``-DSMARTREDIS`` flag.
+An example of this is shown below.
```

Comment:
Might mention that the default is to look for SmartRedis in a directory parallel to the scaling studies

Reply:
good idea i agree